core/domain/exp_services.py

```diff
@@ -407,6 +407,10 @@ def apply_change_list(exploration_id, change_list):
                         state_domain.WrittenTranslations.from_dict(
                             change.new_value))
                     state.update_written_translations(written_translations)
+            elif change.cmd == exp_domain.CMD_ADD_TRANSLATION:
+                exploration.states[change.state_name].add_translation(
```

Comment:
I don't see content_html used here, but it is listed as a required parameter. Is it needed?

Reply:
I'm not sure I understand this completely. Can you please explain or give a line of code where it is used?

Reply:
You can check [here](https://github.com/oppia/oppia/pull/7513/files#diff-1d286750bff5aba07f586681357b8024R77).

Should I move this check to add_translation?